Desktop/TipWindow.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Diagnostics;
+using System.Drawing;
+using System.Text;
+using System.Windows.Forms;
+
+namespace SmartSaver.Desktop
+{
+    
+    public partial class TipWindow : Form
+    {
+
+        public TipWindow()
+        {
+            InitializeComponent();
+        }
+
+        public static string[] tipNames =
+           {
+                "15 Practical Budget Tips",
+                "Basic Budget Tips Everyone Should Know",
+                "8 Simple Ways To Save Money",
+                "7 Tips For Effective And Stress-Free Budgeting",
+                "What Is The 50/20/30 Budget Rule?",
+                "10 Best Ways To Save Money",
+                "10 Budgeting Tips That Really Work"
+            };
+
+        public static string[] tipLinks =
+           {
+                "https://www.daveramsey.com/blog/the-truth-about-budgeting",
+                "https://www.thebalance.com/budgeting-101-1289589",
+                "https://bettermoneyhabits.bankofamerica.com/en/saving-budgeting/ways-to-save-money",
+                "https://www.forbes.com/sites/robertberger/2015/07/26/7-tips-for-effective-and-stress-free-budgeting/",
+                "https://www.investopedia.com/ask/answers/022916/what-502030-budget-rule.asp",
+                "https://www.regions.com/Insights/Personal/Personal-Finances/budgeting-and-saving/10-Best-Ways-to-Save-Money",
+                "https://www.solveyourdebts.com/blog/10-budgeting-tips-that-really-work/"
+            };
```

Comment:
Make separate class Tip with fields for reuse purposes

Reply:
Would that be helpful? I don’t think we’ll use it anywhere else. Maybe I am wrong...

Reply:
First of all, it's OOP meaning you operate with objects, the object represents one entity. The way you coded it here is that you use 2 objects (string arrays) and then somehow merge them together. It's not OOP pattern.

For example, later on we will want to add a short description to a tip, so with this structure you'll have to create other list and make sure that each title, link, and descrition is linked together correcty.